### PR TITLE
Update OrientDB to 2.1-rc2

### PR DESCRIPTION
**Pull request by mcculls**
http://orientdb.com/released-orientdb-2-1-rc2/
http://bamboo.s/browse/NX3-OSSF254-1 :white_check_mark:

### CODE REVIEW

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/model/CUserRoleMappingEntityAdapter.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
 {
   public static final String DB_CLASS = new OClassNameBuilder()
       .prefix("security")
-      .type("user-role-mapping")
+      .type("user_role_mapping")
```
Comment (kellyrob99) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/model/CUserRoleMappingEntityAdapter.java]: dashes no longer allowed I take it?

**Comment by kellyrob99** [comment]
+1

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/model/CUserRoleMappingEntityAdapter.java (hunk 1 above)
Comment (mcculls) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/model/CUserRoleMappingEntityAdapter.java]: correct, part of OrientDB 2.1.x is a more strict SQL parser: https://groups.google.com/forum/#!topic/orient-database/a8TVNVmas4U

**Comment by cstamas** [comment]
+1

thanks (phone),
~t~

**Comment by mcculls** [comment]
Manually merged under 85bb084

**Pull request closed without merging**